src/components/pages/doc.yml

```diff
@@ -11,5 +11,20 @@ commentsPanelTitle:
   en: You are viewing the document page in "comments" mode
   de: Du befindest dich im Kommentar-Modus
 commentsHeader:
-  en: Comment topics
-  de: Kommentar-Themen
+  en: Comments
+  de: Kommentare
+viewHistoryRestrictionTooltip_annonymousUser:
+  en: Viewing document history is only allowed for logged in users
+  de: Ansehen von vorherige Versionen ?? is nur für angemeldet Benutzern elaubt
```

Comment:
Maybe we could incentive the user to login here:

Log in in order to view all versions of this document
Melde dich an, um die alle Versionen dieses Dokuments zu sehen

Then on a second thought, why do we not allow anonymous users to see all versions (they can still go to each version by permalink). The only thing they should not see when not logged in is the restore button :thinking:

Reply:
> Maybe we could incentive the user to login here:
> 
> Log in in order to view all versions of this document Melde dich an, um die alle Versionen dieses Dokuments zu sehen
> 
> Then on a second thought, why do we not allow anonymous users to see all versions (they can still go to each version by permalink). The only thing they should not see when not logged in is the restore button thinking

Sorry, wrong German:
Melde dich an, um **~die~** alle Versionen dieses Dokuments zu sehen

Reply:
True, why not allow history navigation? Let's do that! I will adjust.

Reply:
Change, please have a look 🙏

Reply:
Gorgeous :heart_eyes: